docs/usage.md

```diff
@@ -1,61 +1,94 @@
-# differentialabundance/differentialabundance: Usage
+# nf-core/differentialabundance: Usage
+
+## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/differentialabundance/usage](https://nf-co.re/differentialabundance/usage)
 
 > _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+Differential analysis is a common task in a variety of use cases. In essence, all these use cases entail taking an input matrix containing features (e.g. genes) and observations (e.g. samples), and comparing groups of observations in all or a subset of the features. The feature/ observation language here reflects our hope that this workflow will extend in future to encompass a variety of applications where an assumption of gene vs sample may not be a valid one- though that is the application to which the first release will apply.
+
+With the above in mind, running this workflow requires:
 
-## Samplesheet input
+- a matrix of quantifications with observations by column and features by row
+- a description of the observations such as a sample sheet from RNA-seq analysis
+- a description of the features, for our initial RNA-seq application this is simply the GTF file from which gene annotations can be derived
+- a specification of how the matrix should be split, and how the resulting groups should be compared
 
-You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.
+## Observations (samplesheet) input
 
 ```bash
 --input '[path to samplesheet file]'
 ```
 
-### Multiple runs of the same sample
+This may well be the same sample sheet used to generate the input matrix. For example, in RNA-seq this might be the same sample sheet, perhaps derived from [fetchngs](https://github.com/nf-core/fetchngs), that was input to the [RNA-seq workflow](https://github.com/nf-core/rnaseq). It may be necessary to add columns that describe the groups you want to compare.
```

Comment:
```suggestion
The format is very similar to the samplesheet used for [nf-core/RNA-seq](nf-co.re/rnaseq) or as the output of [nf-core/fetchngs](nf-co.re/fetchngs). It may be necessary to add columns that describe the groups you want to compare.
```

Reply:
This is not what I meant. I meant that this will likely be the same sheet, not just the same format.